src/main/java/teammates/ui/controller/InstructorFeedbackRemindAction.java

```diff
@@ -16,11 +17,19 @@ protected ActionResult execute() {
             nextUrl = Const.ActionURIs.INSTRUCTOR_FEEDBACK_SESSIONS_PAGE;
         }
 
+        FeedbackSessionAttributes feedbackSession = logic.getFeedbackSession(feedbackSessionName, courseId);
+
         gateKeeper.verifyAccessible(
                 logic.getInstructorForGoogleId(courseId, account.googleId),
-                logic.getFeedbackSession(feedbackSessionName, courseId),
+                feedbackSession,
                 false, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
+        if (!feedbackSession.isOpened()) {
+            statusToUser.add(new StatusMessage(
+                    Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSESSIONNOTOPEN, StatusMessageColor.DANGER));
+            return createRedirectResult(nextUrl);
```

Comment:
@leeyimin What should be the status to admin here? Would "Reminder email not sent out to the selected users as the feedback session is not open." be okay?